src/Sulu/Bundle/TestBundle/Testing/KernelTrait.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\TestBundle\Testing;
+
+use Sulu\Component\HttpKernel\SuluKernel;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+trait KernelTrait
+{
+    /**
+     * @var ContainerInterface
+     */
+    protected static $container;
+
+    protected static function getSuluContext(): string
+    {
+        return SuluKernel::CONTEXT_ADMIN;
+    }
+
+    protected static function createKernel(array $options = []): SuluKernel
```

Comment:
Shouldn't we return a `SuluTestKernel` here, as it was done before?

Reply:
No in projects you don't use a custom Test Kernel you use the Project Kernel.